src/transfers.rs

```diff
@@ -49,6 +53,7 @@ fn generate_transfers_from_sp(
     max_distance: f64,
     walking_speed: f64,
     waiting_time: u32,
+    mut need_transfer: Option<NeedTransfer>,
```

Comment:
Why would this mutate along the transfers build?
```suggestion
    need_transfer: Option<NeedTransfer>,
```

Reply:
removed